docs/using/downloading.md

```diff
@@ -0,0 +1,232 @@
+browser window by clicking the **cloud download icon** in the File pane. 
+
+![Downloading Individual Products](../images/download-vertex-single.png "Download Single On Demand Product")
+
+To view the download queue, click the **Downloads** icon in the top band of the Vertex web interface. 
+The Download Queue interface provides options for direct download of individual items along with bulk download options.
+
+### Downloading Multiple Products
+
+You can add products to the Download Queue one by one using the cart icon, or you can search your On Demand 
+products for a specific set of products and add them all to the Download Queue using the **Queue** button at the 
+top of the results list. 
+
+![Downloading Multiple Products](../images/download-vertex-multiple.png "Download Multiple On Demand Products")
+
+There are a number of options available for filtering your On Demand products, with the most useful being the 
+**Project Name** field. You can assign a project name when submitting jobs for processing to easily group items 
+together that are used for the same project. 
+
+Once you've added products to the Download Queue, either by using the individual cart icons or the bulk Queue 
+button, click the **Downloads** icon in the top band of the Vertex web interface. When you open the Download Queue, 
+you have the option to launch direct downloads of individual items in the list, or you can choose to remove 
+individual items from the queue. 
+
+To download all of the products listed in the Download Queue, click the **Data Download** button at the bottom of 
+the queue window and choose from the available options. 
+
+The most robust approach for downloading very long lists of products is the 
+**Download Python Script** option. This downloads a python script that you can launch on your computer. 
+
+- You will be prompted for your Earthdata Login credentials, then the script will work through the list of 
+  download URLs, downloading them one by one until all of the items have been downloaded. 
+- If the script is interrupted during the download, you can simply re-run the same script; it will recognize 
+  any products that have already been successfully downloaded and continue with the remaining items. 
+- To use this option, you must have a Python installation available on your computer. 
+
+Chrome users may find the **Download All** option useful. This option takes advantage of the multi-threading  
+capability in Chrome to download several items at a time.
+
+You can also click the option to **Copy URLs**, which you can then paste into your own download script, 
+if you'd prefer. 
+
+## Programmatic Access
+
+The [HyP3 API](#accessing-products-using-the-hyp3-api) and the 
+[HyP3 Python SDK](#accessing-products-using-the-hyp3-python-sdk) provide programmatic access to On Demand products. 
+To look up your On Demand jobs, you will need to have a valid Earthdata Login (asf-urs) session cookie, 
```

Comment:
```suggestion
To look up your On Demand jobs with the HyP3 API, you will need to have a valid Earthdata Login (asf-urs) session cookie, 
```

Reply:
addressed in [this commit](https://github.com/ASFHyP3/hyp3-docs/pull/590/commits/e48ef007e898cb0d0eead62732faba79c9ce3ff7)